packages/flutter/lib/mountain_view.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Flutter widgets implementing base Android UI components.
+///
+/// To use, import `package:flutter/mountain_view.dart`.
+
```

Comment:
I'd probably add some more docs here explaining how this is different from material.  Also, remove the blank line between the docs and the library directive.